pkg/kobject/kobject.go

```diff
@@ -67,8 +67,6 @@ type ServiceConfig struct {
 	Network       []string            `compose:"network" bundle:"Networks"`
 	Labels        map[string]string   `compose:"labels" bundle:"Labels"`
 	Annotations   map[string]string   `compose:"" bundle:""`
-	CPUSet        string              `compose:"cpuset" bundle:""`
-	CPUShares     int64               `compose:"cpu_shares" bundle:""`
```

Comment:
Please don't remove these, as you can see in the "unsupported keys" section in compose.go, we still grab these from the docker-compose file for (possible) future integration!

>